### PR TITLE
Add accordion and fix minor issues

### PR DESCRIPTION
**Pull request by SkillGG**
- Add accordion
- Fix fetch function to allow removing headers
- Minor web-test UI tweaks
- Add error catching for all web-test functions
- ~fix novelupdates (a bit)~
- fix Syosetu (waiting for separate search and popularNovels filters)

### CODE REVIEW

**Comment by nyagami** [comment]
Could you merge the current `master` to reduce the number of changes

**Review by nyagami — APPROVED** [review]

File: libs/fetch.ts [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
     method?: string;
     body?: FormData | string;
-    [x: string]: string | Record<string, string> | undefined | FormData | Headers;
+    [x: string]:
+        | string
+        | Record<string, string | undefined>
+        | undefined
+        | FormData
+        | Headers;
 };
```
[the remark this answers is not in the document]
Comment (SkillGG) [reply libs/fetch.ts:11]: fetchApi also adds the Cookie tho

**Comment by nyagami** [comment]
Novel Updates doesnt work for me in web

**Comment by SkillGG** [comment]
> Novel Updates doesnt work for me in web

I'll revert the changes to pre-accordion then.

**Review by nyagami — APPROVED** [review]

**Pull request merged**